src/main/resources/model/library.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "library",
+  "eventBodyJavaClass": "org.folio.search.model.dto.locationunit.LibraryDto",
+  "reindexSupported": false,
+  "fields": {
+    "id": {
+      "index": "keyword_lowercase",
+      "showInResponse": [ "search" ]
+    },
+    "tenantId": {
+      "index": "keyword_lowercase",
+      "showInResponse": [ "search" ],
+      "isTenant": true
+    },
+    "name": {
+      "index": "keyword_lowercase",
+      "showInResponse": [ "search" ]
+    },
+    "code": {
+      "index": "keyword_lowercase",
+      "showInResponse": [ "search" ]
+    },
+    "campusId": {
+      "index": "keyword_lowercase"
+    }
```

Comment:
All location-unit models will receive additional metadata field comprised of createdDate & updatedDate

Reply:
Do we really need them? 
cc @SerhiiNosko

Reply:
metdata for locations is introduced in this PR https://github.com/folio-org/mod-search/pull/601/files, so we added also it in the related to Location entities to have the consistent approach. I think it is not bad to have metadata in these entities especially if need search capabilities by metadata fields in future